### ISSUE TITLE
README.md's link to ClusterIT.java is busted

### ISSUE THREAD

**Issue by sming** [spotify/heroic#671]
See https://github.com/spotify/heroic/blob/master/rpc/jvm/README.md :

> For an example of this, see the ClusterIT test suite.

The link is to https://github.com/spotify/heroic/blob/master/heroic-dist/src/test/java/com/spotify/heroic/ClusterIT.java which is nonexistant.

**Comment by sming** [comment]
@mnicholson, any idea which file it should point to? perhaps https://github.com/spotify/heroic/blob/master/heroic-dist/src/test/java/com/spotify/heroic/AbstractLocalClusterIT.java ?

**Comment by sming** [comment]
🏷️ documentation

**Comment by lmuhlha** [comment]
Yup should be the latter, I made a PR: https://github.com/spotify/heroic/pull/672/
